tests/integration/update.bats

```diff
@@ -76,6 +78,10 @@ function check_cgroup_value() {
     [ "$status" -eq 0 ]
     check_cgroup_value $CGROUP_BLKIO "blkio.weight" 500
 
+    # update blkio-leaf-weight
+    runc update test_update --blkio-leaf-weight 500
+    check_cgroup_value $CGROUP_BLKIO "blkio.leaf_weight" 500
```

Comment:
@cyphar , currently, the CI tests are wrote with shell framework. If i add the new flags, some tests are not passed due to the "device" problem. I have tried to add the new flags to CI tests by using 'conditional' judgement, but if do so, the code seems very ugly. For example, if i add new flags into 'DATA; section, i will double the 'old flags' twice. I think this is not acceptable. Can I add those new flags into tests later in another PR if the test framework become more flexible? 
Welcome your suggestions.